src/content/docs/en/guides/deploy/zeabur.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Deploy your Astro Site to Zeabur
+description: How to deploy your Astro site to the web on Zeabur.
+type: deploy
+i18nReady: false
```

Comment:
I think that this should be set to true.
```suggestion
i18nReady: true
```

Reply:
Thanks! Sorry that I thought it should be false until every language is finished :D